src/createUseAdminConfig.ts

```diff
@@ -0,0 +1,50 @@
+import { InjectedProps, Config, ResolvedConfigValue, AdminFields } from "./types";
+import { useCallback, useMemo } from "react";
+import { useWatchLocalstorageEvents } from "./utils";
+
+export function createUseAdminConfig<T extends Record<string, Config>, TNamespace extends string>(
```

Comment:
To clarify the `T` type, could we rename `T` to `TConfig` or `TNamespacedConfig` here? It will also align it with `TNamespace`

Reply:
or is it rather a `TSchema` ?